zendesk/zendesk_client_service.go

```diff
@@ -101,6 +105,19 @@ func NewEnvClient(middleware ...MiddlewareFunction) (Client, error) {
 
 // NewClient creates a new Client.
 //
+
+// // NewZendeskEnvClient create zendesk client for request
+func NewEnvClient() (zClient *client, error) {
+	if zClient == nil {
+		c, err := zendesk.NewClient(util.Config("zendesk.domain"), util.Config("zendesk.username"), util.Config("zendesk.password"))
```

Comment:
hmm this won't work. util comes from plib zendesk is indepandant of plib. function should take all these as an argument

Reply:
This will be handled before the merge.